cypress/e2e/homepage.e2e.ts

```diff
@@ -92,11 +92,11 @@ describe("Homepage integration tests", () => {
   describe("using the nav bar", () => {
     beforeEach(() => {
       cy.visit("/")
-      cy.get("div.navbar-menu > a:nth-child(1)").as("NavBarRentButton")
-      cy.get("div.navbar-menu > a:nth-child(2)").as("NavBarBuyButton")
-      cy.get("div.navbar-menu > a:nth-child(3)").as("NavBarFavoritesButton")
-      cy.get("div.navbar-menu > a:nth-child(4)").as("NavBarAssistanceButton")
-      cy.get("div.navbar-menu > a:nth-child(5)").as("SignInButton")
+      cy.get("div.site-header__navbar-menu > a:nth-child(1)").as("NavBarRentButton")
+      cy.get("div.site-header__navbar-menu > a:nth-child(2)").as("NavBarBuyButton")
+      cy.get("div.site-header__navbar-menu > a:nth-child(3)").as("NavBarFavoritesButton")
+      cy.get("div.site-header__navbar-menu > a:nth-child(4)").as("NavBarAssistanceButton")
+      cy.get("div.site-header__navbar-menu > a:nth-child(5)").as("SignInButton")
     })
```

Comment:
Bloom UIC updated their classNames

Reply:
I wonder if there's a less brittle way to verify these links are present

Better: There are data-test-id's for each of them which should be better than using selectors. 
Best: Use a combination of `findByRole()` and `within()` to better mimic how an end user would look for say the 'Get Assistance' link within the nav bar

Reply:
Good callout! I implemented this in [a new PR](https://github.com/SFDigitalServices/sf-dahlia-web/pull/1788/files).